pandas-stubs/io/parsers/readers.pyi

```diff
@@ -67,6 +67,7 @@ def read_csv(
     infer_datetime_format: bool = ...,
     keep_date_col: bool = ...,
     date_parser: Callable = ...,
+    date_format: str | Mapping[str, str] | None = ...,
```

Comment:
This should be `Mapping[int | str, str]` since an integer column number is accepted.